node-data/src/message.rs

```diff
@@ -457,7 +421,7 @@ pub enum Payload {
     GetInv(payload::Inv),
     GetBlocks(payload::GetBlocks),
     GetData(payload::GetData),
-    CandidateResp(Box<payload::CandidateResp>),
+    CandidateResp(Box<payload::GetCandidateResp>),
```

Comment:
This should be `GetCandidateResp`

Reply:
True

Reply:
This Payload name should also be `GetCandidateResp`